hack/update-generated-runtime-dockerized.sh

```diff
@@ -24,23 +24,16 @@ set -o pipefail
 KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
 KUBE_REMOTE_RUNTIME_ROOT="${KUBE_ROOT}/staging/src/k8s.io/cri-api/pkg/apis/runtime/"
 source "${KUBE_ROOT}/hack/lib/init.sh"
+source "${KUBE_ROOT}/hack/lib/protoc.sh"
+
+kube::protoc::check_protoc
```

Comment:
I'd avoid this and just repeat the `if [[ -z "$(which protoc)" || "$(protoc --version)" != "libprotoc ${PROTOC_VERSION}"* ]]; then` check

I had to do surgery to avoid redefining readonly variables in the scripts in https://github.com/kubernetes/kubernetes/pull/115243/ we probably don't want to repeat in backports